themes/bootstrap3/templates/Recommend/SideFacetsDeferred.phtml

```diff
@@ -1,5 +1,6 @@
 <?php
   $this->headScript()->appendFile('vendor/jsTree/jstree.min.js');
+  $this->headScript()->appendFile('jstree-vufind-facet-plugin.js');
```

Comment:
Is this a good candidate for the `js/lib/` folder?

Reply:
@crhallberg Not sure. If js/lib is for libraries that are somewhat standalone, this is probably an edge case. It's pretty much useless without the custom driving code in facets.js, but it's sort of a standalone plugin.

Reply:
I thought we were working to eliminate the js/lib folder... I don't mind having it if its purpose is clearly defined, but maybe we're better off with more descriptive names if we partition our JS code into directories.